lnbits/core/views/api.py

```diff
@@ -113,6 +113,7 @@ async def api_fiat_as_sats(data: ConversionData):
         return output
 
 
+@api_router.get("/api/v1/qrcode", response_class=StreamingResponse)
 @api_router.get("/api/v1/qrcode/{data}", response_class=StreamingResponse)
 async def img(data):
```

Comment:
can we add a endpoint description here? its not very clear from the code that an url parameter or a query parameter can be used.

```suggestion
async def img(data: str):
```